aten/src/ATen/native/TensorIterator.h

```diff
@@ -122,6 +122,12 @@ struct CAFFE2_API OperandInfo {
 
 struct SplitUntil32Bit;
 
+enum class CommonDTypeStrategy {
```

Comment:
You probably want to `enum class CommonDtypeStrategy : uint8_t` or something to save space.